app-modules/webhook/config/roles/web/webhook_management.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+return [
+    'model' => [
+        'inbound_webhook' => [
+            '*',
```

Comment:
This will ultimately come down to a product decision, but I don't think it should be possible to create/update data that normally originates from an external source. This role should probably be scoped down.

Reply:
I agree, much like the others I mentioned above, this probably only needs view and view-any